src/exporters/cds.py

```diff
@@ -241,7 +242,9 @@ def export(self,
                granularity: str = 'hourly',
                show_api_request: bool = True,
                selection_request: Optional[Dict] = None,
-               break_up: bool = True) -> List[Path]:
+               break_up: bool = True,
+               parallel: bool = True,
+               N_parallel_requests: int = 3) -> List[Path]:
```

Comment:
@tommylees112 , an option here is to not have the boolean parallel field, and only use the n_parallel_requests.
If n_parallel_requests = 1, then it wouldn't run in parallel